pkg/cloud/cloud.go

```diff
@@ -1505,10 +1502,10 @@ func getVolumeAttachmentsList(volume *ec2.Volume) []string {
 }
 
 // Calculate actual IOPS for a volume and cap it at supported AWS limits.
-func capIOPS(volumeType string, requestedCapacityGiB int64, requestedIops int64, minTotalIOPS, maxTotalIOPS, maxIOPSPerGB int64, allowIncrease bool) (int64, error) {
+func capIOPS(volumeType string, requestedCapacityGiB int64, requestedIops int64, minTotalIOPS, maxTotalIOPS, maxIOPSPerGB int64, allowIncrease bool) int64 {
```

Comment:
Unfortunately, this would be a breaking change for customers relying on this clamping for misconfigured storage classes.

Consider the misconfigured storage class: 

```
type: "gp3"
iops: "60000"
```

Currently, this storage class would be able to be created by the driver because [IOPS would be clamped down to 16000](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/0c459169dd183c1b1fb27443d562cc0f6579a62b/pkg/cloud/cloud.go#L75C11-L75C11). If we remove this conditional, this same storage class could no longer be used.

Reply:
Not to say we should never do this (I personally think longterm we should), but that this PR is not the venue for it.